src/directory/directory.mjs

```diff
@@ -1306,6 +1326,11 @@ export const directory = {
             route: '/lib-v1/auth/delete_user',
             filters: ['android', 'ios', 'flutter']
           },
+          {
```

Comment:
This has the same path as the previous block with the same title, should the path be different or have only one or the other?

Reply:
yeah so currently `lib` changed the title to `Delete user account` but kept the same path, so basically I'm changing the title to `lib-v1` and keeping the same path too